auth/util/src/main/java/org/wildfly/security/auth/util/RegexNameValidatingRewriter.java

```diff
@@ -36,10 +37,11 @@
      * Construct a new instance.  The pattern is a partial pattern; if the whole string is to be matched, then
      * the appropriate regex anchors should be used.
      *
-     * @param pattern the pattern that the name must match (or not match) in order to be considered valid
+     * @param pattern the pattern that the name must match (or not match) in order to be considered valid (must not be {@code null})
      * @param match {@code true} if the pattern must match, {@code false} if the pattern must not match
      */
     public RegexNameValidatingRewriter(final Pattern pattern, final boolean match) {
+        Assert.checkNotNullParam("pattern", pattern);
```

Comment:
NIT: this.pattern = Assert.checkNotNullParam("pattern", pattern); 
because checkNotNullParam returns the original value after validation.